dashboard/utils.py

```diff
@@ -38,6 +44,7 @@ class MMTrack:
     financial_aid_min_price = None
     financial_aid_max_price = None
     financial_aid_date_documents_sent = None
+    final_grades = {}
```

Comment:
that is what we use right now as backend for the cache, but if we switch to another backend it will mean that we will need to make another roundtrip to the other backend for this data and, if this is the case, I prefer to rely on the database and the automatic django query caching.